cfpb/retirement
# Repurpose the notes as a Spanish translation field.

The change to svg icons broke translations for the last step of the retirement, where users choose a potential retirement age.

The problem is troublesome to fix using the Django translations framework, because in these cases the icons are being added programmatically, which defeats the language swap-out routine.

For this case, translations are simpler (and much easier to maintain) if we create database fields for the translations. Then js and css can do their thing without interrupting translation operations.

For the steps content, this takes translation work out of the template and moves it to the view, where we can deliver the right content based on the request language.

## Testing
To test inside cfgov-refresh
- With a recent db load, activate your refresh env, pull in this branch locally (in this example, retirement is cloned to /develop-apps/ ), and then install the branch. If you've switched to node 10, you'll probably need to step back to 8 to install retirement, which runs its frontend build.
```bash
nvm use 8
pip install -e ./develop-apps/retirement/
```
- cd to the cfgov-refresh root, run the migrations and load the new content:
```bash
./cfgov/manage.py migrate
./cfgov/manage.py loaddata retiredata.json
```
- fire up `./runserver.sh` and check the Spanish page:
<http://localhost:8000/consumer-tools/retirement/before-you-claim/es/>

Enter values to expose the rest of the page, then scroll to the last step (Paso 3) and step through all the choices at "Elija la edad"

All the content should be translated, unlike the current state on www.
